scripts/release.js

```diff
@@ -156,6 +156,19 @@ logger.warn(`
 
 shouldContinuePrompt();
 
```

Comment:
Can we add some logging here so that when we run this we can see whats happening

Reply:
yes, added in the lasted commit.